engine/syncer.go

```diff
@@ -560,6 +546,13 @@ func (e *ExchangeCurrencyPairSyncer) Start() {
 				continue
 			}
 
+			wsAssetFunctional := exchanges[x].IsAssetWebsocketFunctional(assetTypes[y])
+			if !wsAssetFunctional {
+				log.Warnf(log.SyncMgr,
+					"%s asset type %s websocket functionality has been turned off, REST fetching only.",
```

Comment:
`"%s asset type %s websocket functionality is unsupported, REST fetching only",`